exporter/awsemfexporter/metric_translator_test.go

```diff
@@ -2574,6 +2577,35 @@ func TestTranslateOtToGroupedMetricForInitialDeltaValue(t *testing.T) {
 	}
 }
 
+func TestFetchEntityFields(t *testing.T) {
+	resourceMetrics := pmetric.NewResourceMetrics()
+	resourceMetrics.Resource().Attributes().PutStr(keyAttributeEntityDeploymentEnvironment, "my-environment")
+	resourceMetrics.Resource().Attributes().PutStr(keyAttributeEntityServiceName, "my-service")
+	resourceMetrics.Resource().Attributes().PutStr(attributeEntityNode, "my-node")
+	resourceMetrics.Resource().Attributes().PutStr(attributeEntityCluster, "my-cluster")
+	resourceMetrics.Resource().Attributes().PutStr(attributeEntityNamespace, "my-namespace")
+	resourceMetrics.Resource().Attributes().PutStr(attributeEntityWorkload, "my-workload")
+
+	expectedEntity := cloudwatchlogs.Entity{KeyAttributes: map[string]*string{
```

Comment:
I might be understanding it wrong, shouldn't the entity have the keys in this format?
"KeyAttributes": {
   "Type": "Service",
   "Name": "PetClinic",
   "Environment": "eks:my-cluster"
},
"Attributes": {
    "PlatformType": "AWS::EKS",
    "EKS.Cluster": "MyCluster",
    "K8s.Namespace": "default",
    "K8s.Workload": "customers-service",
    "K8s.Node": "ip-123-213-2139",
    "EC2.InstanceId": "i-0ec9e9bcad8ac3d6d"
}

Reply:
Can you also unit tests covering multiple usecases ? Like Appsignals on EC2, EKS

Reply:
would like to sync up regarding this, so that we are all on the same page. 

Basically, I am under the impression that we should *only* be exporting resource attributes labeled under `aws.entity.<...>`. Right now, there are no labels with that attribute for EC2 tags, so if we need to have those attributes we will need to tag them respectively in the agent.